io/subsystem/src/main/java/org/wildfly/extension/io/BufferPoolResourceDefinition.java

```diff
@@ -55,66 +51,66 @@
     static final RuntimeCapability<Void> IO_POOL_RUNTIME_CAPABILITY =
             RuntimeCapability.Builder.of(IOServices.BUFFER_POOL_CAPABILITY_NAME, true, Pool.class).build();
 
-    private static final int defaultBufferSize;
-    private static final int defaultBuffersPerRegion;
-    private static final boolean defaultDirectBuffers;
+    static final int defaultBufferSize;
```

Comment:
there is no need to store the primitive values as they are wrapped in the ModelNode fields